app/src/main/java/org/fossasia/openevent/app/core/event/create/CreateEventFragment.java

```diff
@@ -280,4 +271,12 @@ private void showLocationLayouts() {
         binding.form.layoutSearchableLocation.setVisibility(View.VISIBLE);
         binding.form.layoutLocationName.setVisibility(View.VISIBLE);
     }
+
+    @Override
+    public void onStop() {
+        super.onStop();
+        ((AppCompatActivity)getActivity()).getSupportActionBar().show();
+        InputMethodManager inputMethodManager = (InputMethodManager) getActivity().getSystemService(INPUT_METHOD_SERVICE);
+        inputMethodManager.hideSoftInputFromWindow(getActivity().getCurrentFocus().getWindowToken(), 0);
```

Comment:
Why this?

Reply:
The soft keyboard was persisting after the event had been created. So used this to hide it.

Reply:
I think there is a util method for it and even so doing it in onStop will hide keyboard on rotation

Reply:
Ok. I'll use the method from utility and add it to the CreateEventFragment's close method.